docs/appendices/release-notes/5.5.1.rst

```diff
@@ -49,4 +49,19 @@ See the :ref:`version_5.5.0` release notes for a full list of changes in the
 Fixes
 =====
 
-None
+- Return meaningful error when trying to drop a column which participates in a
+  :ref:`table level constraint with other columns <check_constraint_multiple_cols>`
+  , or its sub-columns participate in a
+  :ref:`table level constraint <check_constraint_multiple_cols>` together with
+  columns that are not sub-columns of the column to be dropped, For example, we
+  cannot drop column ``o['oo']`` from the following tables::
+
+    CREATE TABLE t1 (a int, o object AS (oa int, oo object AS(ooa int, oob long)),
+                     CHECK (o['oo']['oob'] + o['oa'] > 0))
+    CREATE TABLE t2 (a int, o object AS (oa int, oo object AS(ooa int, oob long)),
+                     CHECK (o['oo']['oob'] + a > 0))
+    CREATE TABLE t3 (a int, o object AS (
+                       oa int,
+                       oo object AS(ooa int, oob long),
+                       obj object AS (obja int, objb long))
+                     CHECK (o['oo']['oob'] + o['obj']['objb'] > 0))
```

Comment:
I think this could be simplified and still addresses all cases. 
```suggestion
- Return meaningful error when trying to drop a column which itself or it's sub-columns participates in a
  :ref:`table level constraint with other columns <check_constraint_multiple_cols>`.
  For example, we  cannot drop column ``my_object`` from the following tables::

    CREATE TABLE t1 (a int, my_object object AS (col_a int)),
                     CHECK (my_object + i > 0))
    CREATE TABLE t2 (a int, my_object object AS (col_a int)),
                     CHECK (my_object['col_a'] + i > 0))
```

Reply:
Maybe you mean change `a` to `i`? also `my_object + i > 0` doesn't work, cannot add to an object :)

Reply:
> Maybe you mean change `a` to `i`? also `my_object + i > 0` doesn't work, cannot add to an object :)

Yes sure, was just a typo :D

Reply:
ok, thx!